jobs/process_unassigned_chunks.go

```diff
@@ -384,8 +384,8 @@ func HandleTreasureChunks(chunks []oyster_utils.ChunkData, session models.Upload
 
 	for i := 0; i < len(chunks); i++ {
 		if _, ok := treasureMap[int(chunks[i].Idx)]; ok {
-			address := make([]giota.Address, 0, 1)
-			chunkAddress, err := giota.ToAddress(chunks[i].Address)
+			address := make([]trinary.Hash, 0, 1)
```

Comment:
This might be another place to look at using a different type, if applicable.  

The reason is because iota treats hashes and addresses as different things.  Both are ultimately 81 character tryte strings, but an address is where a transaction is located on the tangle, while a hash is derived from the overall transaction trytes.  So every transaction has an address and a hash, but they are different for each transaction.  

It doesn't matter that much since both are 81 character tryte strings, and it's possible that iota has decided that hashes and addresses are so similar to just use the same type for both.  We should check.

Reply:
Correct,

See here:
https://github.com/iotaledger/iota.go/blob/5ef833c00afc210d70375c2f25e86be6a9ff6b9b/transaction/transaction.go#L22
Both Hash and Address has the same type

Reply:
Okay sounds good.  Disregard the trytes/address/hash comments.